src/styles.css

```diff
@@ -25,5 +25,13 @@ html {
   @apply flex-grow
 }
 
+.article-content {
```

Comment:
We need a consistent way of handling stylesheets, there are currently two separate ones for no specific reasons. I would say that we can either build a proper component stylesheet that is reused consistently across the page, or we continue with inline styling like most of the components have so far. This might interest also @Bruck1701.
Let me know what you guys think!

Reply:
Totally agree that we should try to keep everything as consistent as possible! 
Even though I used inline styling, I think it would be better to have a component stylesheet. 
Inline styling can make it harder to change the code uniformly later, right?